sdk/storage/azure-storage-queue/tests/test_queue_client.py

```diff
@@ -341,6 +341,43 @@ def test_create_service_with_conn_str_succeeds_if_sec_with_primary(self, resourc
             self.assertTrue(service.primary_endpoint.startswith('https://www.mydomain.com/'))
             self.assertTrue(service.secondary_endpoint.startswith('https://www-sec.mydomain.com/'))
 
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(name_prefix='pyacrstorage')
+    def test_create_service_with_custom_account_endpoint_path(self, resource_group, location, storage_account, storage_account_key):
+        custom_account_url = "http://local-machine:11002/custom/account/path/" + self.sas_token
```

Comment:
Can we double-check the test coverage that we have for .NET (assuming that this is the most complete)?

Reply:
.Net doesn't appear to have much test coverage for the custom "BlobEndpoint" scenario....
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Common/tests/StorageConnectionStringTests.cs